spec/controllers/social_networking/comments_controller_spec.rb

```diff
@@ -17,14 +19,13 @@ module SocialNetworking
         participant: participant,
         text: "I like cheeses",
         item_id: 5,
-        item_type: "SocialNetworking::OnTheMindStatement")
+        item_type: "SocialNetworking::OnTheMindStatement"
```

Comment:
Sure - the point of this PR is not to refactor though. I just wanted to get specs passing against Rails 5. If I refactored at the same time and things broke it would be a lot harder to figure out the source of the problem.